packages/@aws-cdk/aws-apigatewayv2/README.md

```diff
@@ -222,6 +223,44 @@ with 3 API mapping resources across different APIs and Stages.
 | api | beta  |   `https://${domainName}/bar`  |
 | apiDemo | $default  |   `https://${domainName}/demo`  |
 
+### Managing access
+
+API Gateway supports multiple mechanisms for [controlling and managing access to your HTTP
+API](https://docs.aws.amazon.com/apigateway/latest/developerguide/http-api-access-control.html) through authorizers.
+
+#### JWT Authorizer
```

Comment:
Update and move this section to the authorizers package. Keep a brief explanation on authorizers and refer to the authorizers package for more details.